00_Installation/requirements.txt

```
@@ -1,7 +1,7 @@
 scipy>=1.7.1
 photutils>=1.1.0
 scikit-image>=0.16.2
-astropy==4.3.1
+astropy>=4.3.1
```

```suggestion
astropy<5.0
```

Astropy 5.0 causes the imports in (at least) the Background Sky notebook to fail.